src/System.Private.ServiceModel/tests/Common/Infrastructure/xunit/SkippedTestCase.cs

```diff
@@ -0,0 +1,53 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+
+namespace Infrastructure.Common
+{
+    /// <summary>Wraps another test case that should be skipped.</summary>
```

Comment:
I was going to but @hongdai and I decided that this class will be renamed and diverge from BuildTools to be more extensible.  Actually, I think SkippedTestCase in BuildTools was copied from an xunit sample anyway.